### ISSUE TITLE
Support Keyboard

### ISSUE THREAD

**Issue by CalebKester** [lokyoung/vuejs-paginate#24]
Paginate doesn't respect keyboard navigation. It would be great to support accessibility. When I have a link focused hitting enter should trigger the pagination.

**Comment by lokyoung** [comment]
Hello @CalebKester . Could you please provide more detailed requirements? Thanks.

**Comment by CalebKester** [comment]
https://jsfiddle.net/lokyoung/u3u3nzns/

I can click on 1,2,3,4,5 and it behaves like it's supposed to. However when I use keyboard navigation and tab through the links I should be able to trigger the click by pushing the "enter" key. I can screen record a video or you can take a look at how google does it. Just hit tab a bunch until you get to their pager, then hit enter to activate the pager link.

**Comment by lokyoung** [comment]
Hello @CalebKester . Thanks for your feedback. I have added a PR for solve this issue and publish a new npm version. You can check `v0.8.3` to see if it meets your requirement. Please feel free to ask me if any problems.

**Comment by lokyoung** [comment]
@CalebKester Use new release `v0.8.4`. Thanks.

**Comment by CalebKester** [comment]
Looks really good! Thanks for getting that fix out so fast!
